src/main/java/de/uni_tuebingen/ub/nppm/db/EinzelbelegDB.java

```diff
@@ -35,4 +39,55 @@ public static Einzelbeleg getFirstPublicEinzelbeleg() throws Exception {
     public static void insertBySql(String sql) throws Exception {
         insertOrUpdate(sql);
     }
+
+    //Neu Dazu gekommen -- Lemma
+    public static boolean hasZusatzNamenKommentar(String lastID) throws Exception {
+        try ( Session session = getSession()) {
+            String SQL = "SELECT * FROM einzelbeleg_hatnamenkommentar WHERE EinzelbelegID = :lastID";
+
+            if (session.createNativeQuery(SQL) != null) {
+                NativeQuery query = session.createNativeQuery(SQL);
+                query.setParameter("lastID", lastID);
+                return !query.getResultList().isEmpty();
+            } else {
+                return false;
+            }
+        } catch (Exception e) {
+            return false;
+        }
+    }
+
+    public static boolean hasLemmar(String lastID) throws Exception {
```

Comment:
hasLemmar => hasLemma

Reply:
and this should as well be implemented as utility function in the Einzelbeleg Model class

Reply:
done